repository/auth_repo.go

```diff
@@ -63,6 +93,24 @@ func (a AuthRepo) Get(ctx context.Context, key domain.AuthAPIKey) (string, bool)
 	return string(environment), true
 }
 
+// GetAll gets all values from auth repo
+func (a AuthRepo) GetAll(ctx context.Context) (map[domain.AuthAPIKey]string, bool) {
```

Comment:
Minor thing but if this method is only being called by the AuthRepo we could make it private?

Reply:
Sorted